### ISSUE TITLE
Can't run tests locally when using zsh

### ISSUE THREAD

**Issue by Yelwo** [apptension/aws-boilerplate#48]
# Steps to reproduce 
```
zsh
make shell
cd /services/workers
make test
```

# Encountered behavior
```
imx_501_db_1 is up-to-date
bash ./scripts/run_tests.sh
All done! ✨ 🍰 ✨
25 files would be left unchanged.
./scripts/wait-for-it.sh: line 146: realpath: command not found
./scripts/wait-for-it.sh: line 58: timeout: command not found
wait-for-it.sh: timeout occurred after waiting 15 seconds for localhost:5432
make[1]: *** [test] Error 127
```
# Expected behavior
 make test run without error

# Workaround fix
* Change shell to bash
```
make shell USER_SHELL=bash
```
or:
* Install coreutils before running tests
```
brew install coreutils
```

**Comment by pziemkowski** [comment]
Related to #53 . Moving tests to docker would fix both issues

**Comment by pziemkowski** [comment]
Resolved in #55
